massgov/openmass
# DP-23084: A11y don't render empty link tag for org key message with no link

**Description:**
Updated the Featured Message paragraph template to conditionally show the callout link.


**Jira:** (Skip unless you are MA staff)
DP-23084


**To Test:**
- Go to /orgs/executive-office-of-health-and-human-services.
- Inspect the Featured Message code.
- Verify there isn't an empty <a> tag with the class "ma__callout-link".
- Go to /orgs/massachusetts-registry-of-motor-vehicles.
- Verify the callout link is displayed.


---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
